jetstream/bigquery_client.py

```diff
@@ -112,3 +112,23 @@ def touch_tables(self, normandy_slug: str):
     def delete_table(self, table_id: str) -> None:
         """Delete the table."""
         self.client.delete_table(table_id, not_found_ok=True)
+
+    def delete_experiment_tables(
```

Comment:
It seems like this is a somewhat dangerous operation that might be worth adding a test, at least to validate that the regex is including/excluding everything we expect?

Reply:
I added some tests